lib/build-view.js

```diff
@@ -56,8 +57,20 @@ module.exports = (function() {
     if (this.panel) {
       this.panel.destroy();
     }
-    this.panel = atom.workspace.addBottomPanel({ item: this });
-    this.height = this.output.offset().top + this.output.height();
+
+    if ('Top' === atom.config.get('build.panelOrientation')) {
```

Comment:
I think a switch-case would be more appropriate here.

Also, I think the default value should be bottom - to keep backward compatability. This is also likely the cause of all the broken tests.